avionics/cmake-build-debug/FLARE.cbp

```diff
@@ -0,0 +1,891 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Was this file supposed to be added? `cmake-build-debug` sounds like temporary files

Reply:
Nope that's my bad.  I'll fix that